msteams-platform/bots/how-to/conversations/send-proactive-messages.md

```diff
@@ -54,11 +54,28 @@ Create the conversation, after you have the user or channel information.
 
 ## Create the conversation
 
-Create the conversation if it doesn't exist or you don't know the `conversationId`. Create the conversation only once and store the `conversationId` value or `conversationReference` object.
+You can create the conversation if it doesn't exist or you don't know the `conversationId`. Create the conversation only once and store the `conversationId` value or `conversationReference` object.
+
+To create a conversation, you need `userId`, `tenantId`, and `serviceUrl`.
+
+For `serviceUrl`, use the value from an incoming activity triggering the flow or one of the global service URLs. The following is an example for the `serviceUrl`:
+
+```csharp
+const serviceUrl = 'https://smba.trafficmanager.net/uk/';
+        `MicrosoftAppCredentials.trustServiceUrl(serviceUrl);
+```
+
+If the `serviceUrl` isn't available from an incoming activity triggering the proactive scenario, use the following global URL endpoints:
+
+* Production: `https://smba.trafficmanager.net/teams/`
```

Comment:
@tacri Do we call this "Production", or "Public"? Just wanted to be precise with the terminology.